src/test/java/jalau/at18/katas/likes/sarai/MainTest.java

```diff
@@ -0,0 +1,11 @@
+package jalau.at18.katas.likes.sarai;
+
+import static org.junit.Assert.*;
+public class MainTest {
+    public void shouldReturnLikeThis() {
+        String input = "juan lulu jose";
+        String expected = "juan,lulu,jose, like this";
+        assertEquals(expected, input);
+        assertEquals("pepe,mario, like this", "pepe mario" );
```

Comment:
This would fail, but actually, this assert would not make too much sense, you are comparing to constant values instead of an expected value vs result of your code